src/sidebar/components/test/ThreadList-test.js

```diff
@@ -302,6 +316,73 @@ describe('ThreadList', () => {
     assert.equal(cards.length, fakeTopThread.children.length);
   });
 
+  describe('chapter headings', () => {
+    const addThread = (cfi, title) => {
```

Comment:
Optional ergonomic suggestion: rename this function to be more clearly discrete from similarly-named outer `createThread` function, perhaps `addThreadWithSegment`, although that might be clunky. You decide!

Reply:
Makes sense. I renamed this `addThreadInChapter`.